Base/res/words.txt

```diff
@@ -1,3 +1,6 @@
+# Copyright (c) 2022, Peter Forret
+# SPDX-License-Identifier: MIT
+# https://github.com/pforret/shwordle
```

Comment:
I'm not sure Peter Forret has a copyright on a list of English words. :)

Reply:
Fair enough, but that specific list of words was generated by him (I assume) and it was suggested that I put an attribution in the list when I asked on discord. Maybe changing it from copyright to just indicate that the list of words is from his repository?

Reply:
Maybe make your own list of words? It can start small. We try not to import existing stuff, for the most part.

Reply:
I'm not opposed to that idea, but in the long term, I imagine Serenity will eventually have a spell checking lib (or similar) that depends on a word list. Ideally, I would use this file, however I don't think there is such a thing in Serenity as of yet? However, when the time comes, surely the word list isn't going to be crafted by hand and memory. According to [wikipedia](https://en.wikipedia.org/wiki/Words_(Unix)), linux/unix words are sourced from the "Moby Project" -- a public domain list of words. Would it be reasonable for me to use that as a resource?

Reply:
> I'm not sure Peter Forret has a copyright on a list of English words. :)

Technically it would be database rights and not copyright if he compiled the list himself, so it can still be protected. :-)

Reply:
> spell checking lib (or similar) that depends on a word list

I've messed with that before (https://github.com/alimpfard/serenity/commit/b2c249fb2984058f2026a253cda3123ebb1a1a83), and a sanitised list of words scraped from the text in the serenity repo is a fairly decent starting point.

```console
$ find Base Documentation -name '*.md' | xargs grep -hoE '\b[a-zA-Z]+\b' | tr [[:upper:]] [[:lower:]] | sort | uniq > serenity.words.db
```

Reply:
That is very interesting -- thanks for pointing it out! 

Sounds like we had a similar idea for the word list as I also ended up processing the documentation. For the game, it seems to be just fine, and even adds a fun technical twist vs a normal dictionary.